hera_cal/delay_filter.py

```diff
@@ -55,13 +56,12 @@ def run_delay_filter(self, to_filter=None, weight_dict=None, horizon=1., standof
                 time. Skipped channels are then flagged in self.flags.
                 Only works properly when all weights are all between 0 and 1.
             tol : float, optional. To what level are foregrounds subtracted.
-            verbose: If True print feedback to stdout
             cache_dir: string, optional, path to cache file that contains pre-computed dayenu matrices.
                         see uvtools.dspec.dayenu_filter for key formats.
             read_cache: bool, If true, read existing cache files in cache_dir before running.
             write_cache: bool. If true, create new cache file with precomputed matrices
                                that were not in previously loaded cache files.
-            cache: dictionary containing pre-computed filter products.
+            skip_flagged_edges : bool, if true do not include edge freqs in filtering region (filter over sub-region).
```

Comment:
is this only a bool? how do you specify the edge freqs if this is true? if this is mixed type, then please specify

Reply:
If this is true, then the method automatically identifies edges that are fully flagged (e.g. a channel where all times are flagged) and does not include them in the filter. I'm updating the docstring to make this more clear.